flask_app/blueprints/beams.py

```diff
@@ -76,7 +77,7 @@ def get_all():
     },
     'required': ['beam']
 })
-def create(user):
+def create(user: User) -> Union[Response, Tuple[str, int]]:
```

Comment:
This return type seems to be used frequently, add a name for it instead of copying it.